infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -46,13 +56,25 @@ Run `terraform init` when setting up a new configuration, changing backend setti
 
 _If using tfvars_
 `terraform init -backend-config="./backend.tfvars"`
+
 _If using environment variables_
 `terraform init`
 
 ### Planning & Applying
 
 `terraform plan` to see the plan generated by terraform for the specified deployment configuration. If you are satisfied with the plan, run `terraform apply` to apply the configuration.
 
+You will need to run `terraform init`, `terraform plan` and `terraform apply` from two dirrent directories. First, run the commands from the `../container/` directory to set up the Azure Container Registry. Then, run the commands from the `sandbox/` (current) directory to set up the rest of the sandbox.
```

Comment:
```suggestion
#### Docker Images - Publish

Docker container images that are built locally should be published to Azure Container Registry (ACR). You can use the Terraform code provided in [Azure Container Terraform](infra/recipes/terraform/azure/environments/container) to push the images to ACR.

Run the following three commands for ACR publishing

1. `terraform init` : Run it first time to setup tfbackend
2. `terraform plan` : Plan the Terraform deployment
3. `terraform apply` : Apply the Terraform deployment

If you don't want to use terraform, you can always use `docker push` command to publish the container images available in local docker to ACR.

#### Rest of Infra - Setup

As part of bringing OpenHouse in Azure, you will need to setup other Azure services like AKS, MySQL, Storage. You can use the Terraform code provided in [Azure Sandbox Terraform](infra/recipes/terraform/azure/environments/sandbox) to setup rest of infra that is required for successfully bringing up OpenHouse.

Run the following three commands for rest of infra setup

1. `terraform init` : Run it first time to setup tfbackend
2. `terraform plan` : Plan the Terraform deployment
3. `terraform apply` : Apply the Terraform deployment
```

Reply:
We could add the below main terraform command in a separate section and refer in both the places above.

Reply:
I will add this but I just want to clarify about the above comment before -- the images do not need to be built manually, only the jar files

Reply:
I see. Updated accordingly